src/Microsoft.Extensions.Logging/LoggerFactory.cs

```diff
@@ -41,11 +42,35 @@ public LoggerFactory(IConfiguration configuration)
                 throw new ArgumentNullException(nameof(configuration));
             }
 
+            UseConfiguration(configuration);
+        }
+
+        /// <summary>
+        /// Replaces the <see cref="IConfiguration"/> used for filtering.
```

Comment:
I think usage will mostly be `factory.UseConfiguration(config.GetSection("Logging"))`
with the config having:
```json
{
  "OtherStuff": {
  },
  "Logging": {
    "LogLevel": {
        "System": "Trace"
    }
  }
}
```

When would you want to use multiple sections?